src/ui/server/helpers/summary-lists/policy/policy-and-date-fields/index.ts

```diff
@@ -73,7 +73,7 @@ const generatePolicyAndDateFields = (answers: ApplicationPolicy, referenceNumber
         data: answers,
         ...changeLink(answers[POLICY_TYPE], referenceNumber, CURRENCY_CODE, checkAndChange),
       },
-      answers[POLICY_CURRENCY_CODE] && getCurrencyByCode(currencies, answers[POLICY_CURRENCY_CODE]).name,
+      getCurrencyByCode(currencies, answers[POLICY_CURRENCY_CODE]).name,
```

Comment:
would this not break if for any reason the currency comes back as null?

Reply:
It can't be null in the summary list scenarios - to get to the summary list (check your answers), a `POLICY_CURRENCY_CODE` must have been submitted. Otherwise, the section is marked as incomplete and this function will not be called.